src/main/java/sg/ncl/MainController.java

```diff
@@ -1518,7 +1533,7 @@ public String teams(Model model, HttpSession session) {
 //        String teamName = teamManager.getTeamNameByTeamId(teamId);
 //        teamManager.setInfoMsg("You have just joined Team " + teamName + " !");
 //
-//        return "redirect:/teams";
+//        return REDIRECT_TEAMS;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)